serving/docker/partition/sm_neo_utils.py

```diff
@@ -32,7 +32,7 @@ def get_neo_env_vars():
     Get environment variables required by the SageMaker Neo interface
     """
     try:
-        compiler_options = os.environ["COMPILER_OPTIONS"]
+        compiler_options = os.environ.get("COMPILER_OPTIONS")
```

Comment:
please provide a default to getter method

Reply:
Intended behavior is for this to return `None` if no `COMPILER_OPTIONS` are passed since compiler_options will be optional in Neo. dict.get() already returns None by default. Should I make explicit with 
```
.get("COMPILER_OPTIONS", None)
```
?